datanode/sqlstore/migrations/0036_oracle_data.sql

```diff
@@ -0,0 +1,69 @@
+-- +goose Up
+-- First make oracle_data a hypertable
+-- +goose StatementBegin
+DO $$
+BEGIN
+    IF NOT EXISTS (SELECT * FROM timescaledb_information.hypertables WHERE hypertable_name = 'oracle_data') THEN
+        PERFORM create_hypertable('oracle_data', 'vega_time', chunk_time_interval => INTERVAL '1 day', migrate_data => true);
+    END IF;
+END $$;
+
+-- +goose StatementEnd
+-- Now make our new join table to link oracle data to oracle specs
+CREATE TABLE IF NOT EXISTS
+    oracle_data_oracle_specs (
+        vega_time TIMESTAMP WITH TIME ZONE NOT NULL,
+        seq_num BIGINT NOT NULL,
+        spec_id bytea NOT NULL,
+        PRIMARY KEY (vega_time, seq_num, spec_id)
+    );
+
+CREATE INDEX IF NOT EXISTS idx_oracle_data_oracle_specs_spec_to_data ON oracle_data_oracle_specs (spec_id);
+
+-- Make it a hypertable
+SELECT
+    create_hypertable ('oracle_data_oracle_specs', 'vega_time', chunk_time_interval => INTERVAL '1 day');
+
+-- Populate it with the contents of the 'matched_specd_ids' bytea array column
```

Comment:
```suggestion
-- Populate it with the contents of the 'matched_specs_ids' byte array column
```

Reply:
That one is fair enough though it should actually be matched_spec_ids

Reply:
fixed

Reply:
wasn't sure which way it should be, thanks for fixing

Reply:
@pscott31 "bytea" is irrelevant I guess? Or is that how it should be?

Reply:
yes that's the name of the database type; odd as it is